src/create.rs

```diff
@@ -450,6 +480,17 @@ mod tests {
             }
         );
     }
+    #[test]
```

Comment:
Let's have blank lines between tests to ease reading

Reply:
FWIW, I think `rustfmt` will do that.